community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala

```diff
@@ -147,3 +149,12 @@ class CommunityBuildTest {
   //   command = "compile"
   // )
 }
+
+class TestCategory
+class UpdateCategory
+
+@Category(Array(classOf[UpdateCategory]))
+class CommunityBuildUpdate extends CommunityBuildTest {
+  override def test(project: String, command: String): Unit =
+    super.test(project, command.split("/").head + "/update")
```

Comment:
That's not correct for https://github.com/lampepfl/dotty/blob/38189b5a6b4556a94f998676206d4d59b8b09aa5/community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala#L89

`jvm/update` won't download the test dependencies, you need `jvm/test:update` to be complete.
I suggest changing the `test` method to be used like this to be more general:
```scala
  @Test def scalatest = test(
    project = "scalatest",
    testCommand = "jvm/test:compile",
    updateCommand = "jvm/test:update"
)
```
It's a bit more verbose but there's no surprise this way.

Reply:
:+1: